WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md

```diff
@@ -13,9 +13,9 @@ ms.technology: identity-adds
 ---
 # Enabling Clients to Locate the Next Closest Domain Controller
 
->Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
+>Applies To: Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012, Windows Server 2008 R2, Windows Server 2008
```

Comment:
Please remove ", Windows Server 2008 R2, Windows Server 2008" as we no longer refer to them in the tag lines.

Reply:
While I personally don't agree with this considering they still have 18 months of support left, I've updated in line with policy